sdcm/nemesis.py

```diff
@@ -443,13 +443,23 @@ def _terminate_cluster_node(self, node):
         self.cluster.terminate_node(node)
         self.monitoring_set.reconfigure_scylla_monitoring()
 
-    def disrupt_nodetool_decommission(self, add_node=True):
+    def disrupt_nodetool_decommission(self, add_node=True, is_seed_node=False):
         self._set_current_disruption('Decommission %s' % self.target_node)
         self.cluster.decommission(self.target_node)
         if add_node:
             # When adding node after decommission the node is declared as up only after it completed bootstrapping,
             # increasing the timeout for now
             new_node = self._add_and_init_new_cluster_node()
+            if is_seed_node:
+                new_node.is_seed = is_seed_node
+                if new_node not in self.cluster.seed_nodes:
```

Comment:
move this logic out of this function, you can do that in `disrupt_nodetool_seed_decommission`

Reply:
done